### PR TITLE
Adding a few missing traits from auditing against Nethys' trait list.

### PR DESCRIPTION
**Pull request by VestOfHolding**

### CODE REVIEW

File: src/scripts/config/traits.ts [hunk 1]
```diff
@@ -535,6 +535,7 @@ const featTraits = {
     additive1: "PF2E.TraitAdditive1",
     additive2: "PF2E.TraitAdditive2",
     additive3: "PF2E.TraitAdditive3",
+    afterMath: "TraitAftermath",
```
Comment (stwlam) [inline src/scripts/config/traits.ts:538]:
```suggestion
    aftermath: "PF2E.TraitAftermath",
```

File: src/scripts/config/traits.ts [hunk 2]
```diff
@@ -553,6 +554,7 @@ const featTraits = {
     fortune: "PF2E.TraitFortune",
     general: "PF2E.TraitGeneral",
     injury: "PF2E.TraitInjury",
+    instinct: "PF2E.TraitInstinct",
```
Comment (stwlam) [inline src/scripts/config/traits.ts:557]: This was dropped in the second CRB printing

File: static/lang/en.json [hunk 3]
```diff
@@ -3195,6 +3196,7 @@
         "TraitBludgeoning": "Bludgeoning",
         "TraitBoggard": "Boggard",
         "TraitBomb": "Bomb",
+        "TraitBonded": "Bonded",
```
Comment (stwlam) [inline static/lang/en.json:3199]:
```suggestion
```

Comment (VestOfHolding) [reply static/lang/en.json]: Done.

**Pull request merged**